WORKSPACE

```
@@ -2,15 +2,20 @@ workspace(name = "io_tweag_rules_nixpkgs")
 
 # For documentation
 
+local_repository(
+    name = "io_tweag_rules_nixpkgs_docs",
```

With the modularization we also wanted to drop the `io_tweag_` prefix in the individual modules.
```suggestion
    name = "rules_nixpkgs_docs",
```
IIRC it was only still kept in the top-level `io_tweag_rules_nixpkgs` for backward compatibility.
All the other modules, e.g. `rules_nixpkgs_java`, don't use that prefix anymore.